e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/T_TwinBrowseOrchestratedTestTheory.cs

```diff
@@ -0,0 +1,249 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.Orchestrated {
+    using IIoTPlatform_E2E_Tests.TestExtensions;
+    using Newtonsoft.Json;
+    using RestSharp;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
+    [Collection("IIoT Multiple Nodes Test Collection")]
+    [Trait(TestConstants.TraitConstants.PublisherModeTraitName, TestConstants.TraitConstants.PublisherModeOrchestratedTraitValue)]
+    public class T_TwinBrowseOrchestratedTestTheory {
+        private readonly IIoTMultipleNodesTestContext _context;
+
+        public T_TwinBrowseOrchestratedTestTheory(IIoTMultipleNodesTestContext context, ITestOutputHelper output) {
+            _context = context;
+            _context.OutputHelper = output;
+        }
+
+        [Fact, PriorityOrder(0)]
+        public async void SwitchToOrchestratedMode() {
```

Comment:
Please avoid async test methods, the xunit ordering is not fixed when running on build agent